app/views/openid_connect/shared/redirect_js.html.erb

```diff
@@ -10,7 +10,7 @@
           <%= t('saml_idp.shared.saml_post_binding.no_js') %>
         </p>
 
-        <%= link_to(t('forms.buttons.continue'), @oidc_redirect_uri, class: 'usa-button usa-button--wide usa-button--big', data: { click_immediate: '' }) %>
+        <%= link_to(t('forms.buttons.submit.default'), @oidc_redirect_uri, class: 'usa-button usa-button--wide usa-button--big', data: { click_immediate: '' }) %>
```

Comment:
This change is made to bring consistency with the button text on the SAML no-JavaScript screens, and also because the preceding paragraph refers to "submit" button:

>please click the submit button to continue signing in or signing out

This avoids needing to conditionally consider `:oidc` or `:saml` in common test flows, and instead use `click_submit_button` when reaching this step of the flow when JavaScript is disabled.

Reply:
(cc @mitchellhenke in case there's any conflicts to be concerned with here)

Reply:
ah, nope, I don't think so 👍🏼